lib/gpu/gpuCommand.hpp

```diff
@@ -187,6 +188,10 @@ class gpuCommand : public kotekan::kotekanLogging {
     /// State that is shared by instances of this command (at one point in the pipeline)
     std::shared_ptr<gpuCommandState> command_state;
 
+    /// Ensure serial execution of calls to the same instance
+    std::mutex execute_mutex;
+    std::unique_lock<std::mutex> execute_lock;
+
```

Comment:
Do we actually want to keep this after having fixed the race condition?

Reply:
No, I thought I had removed these.